modules/dnn/src/layers/lrn_layer.cpp

```diff
@@ -91,7 +91,7 @@ class LRNLayerImpl CV_FINAL : public LRNLayer
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
         if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
-            return bias == 1;
```

Comment:
what is the problem with `bias == 1`?

Reply:
`bias` is floating point number in general.
IE API expects integers only.

Reply:
@alalek 
thanks for the reply.

I cannot see the point of changing
`bias == 1`
to
`bias == (int)bias`

Why is it related to the constraint of IE API, as it is only a comparison?

Reply:
Code looks good to me:
```
if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
    return bias == (int)bias;  // only integers are supported by IE
... other backends ...
```

Reply:
@csukuangfj, Inference Engine may works with `bias == 1` and `bias == 2` and any other integers. However it cannot manage `bias == 1.1` in example. That is why the condition has been changed to `bias == (int)bias` which is similar to `bias == std::floor(bias)` that means bias is integer.

Reply:
@dkurt 
aha, i see. Thanks!